File: BOJ/완전탐색/1748.py
# 1748 수 이어 쓰기 1
# https://www.acmicpc.net/problem/1748
import sys
def input(): return sys.stdin.readline().rstrip()


N = int(input())
answer = 0
i = 1

while i <= N:
    answer += (N-i+1)
    i *= 10
print(answer)

File: BOJ/구현/1748.cpp
// 1748 수 이어 쓰기 1
// https://www.acmicpc.net/problem/1748
# include <iostream>

using namespace std;

int main(void) {
    int N, answer = 0, i = 1;

    cin >> N;

    while (i <= N)
    {
        answer += (N -i + 1);
        // 15
        // 15 + 6

        i *= 10;
    }

    cout << answer << '\n';
    
    return 0;
}